varia/functions.php

```diff
@@ -225,15 +225,18 @@ function varia_setup() {
 				'enable_theme_default' => true,
 			)
 		);
-		
+
 		// Add support for Content Options.
 		add_theme_support( 'jetpack-content-options', array(
 			'blog-display' => 'content',
 			'featured-images' => array(
 				'archive'         => true,
 				'archive-default' => true,
 				'post'            => true,
+				'post-default'    => true,
```

Comment:
Oh, on second thought - I thought the original issue recommended setting this to false?

```suggestion
				'post-default'    => false,
```

Reply:
I think only setting 'page-default' to false is necessary.

As you mentioned there is no option to turn it off before the Jetpack change... and from what I observe if a Post has a featured image it is displayed, no option otherwise.  So setting the 'post-default' value to false means that a site doesn't change based on this code change.

Reply:
Ah, yes, whoops, I got muddled. Yes, I think this is good.